modules/jaeger-sidecar-automatic.adoc

```diff
@@ -32,6 +32,7 @@ spec:
     spec:
       containers:
       - name: myapp
+      image: acme/myapp:myversion
```

Comment:
Need to indent this by 2 spaces.

Reply:
Weird, it's properly indented under name: on my system.